package.json

```diff
@@ -24,7 +24,9 @@
   "bin": {
     "shjs": "./bin/shjs"
   },
-  "dependencies": {},
+  "dependencies": {
+    "glob": "^6.0.3"
+  },
   "devDependencies": {
```

Comment:
Is it possible to use glob without adding it as a package dependency? I know some of the original maintainers wanted to keep dependencies at a minimum (https://github.com/shelljs/shelljs/issues/122#issuecomment-112498119).